### PR TITLE
Automatically generate password to avoid using Convert-To-SecureString

### PR DESCRIPTION
**Pull request by kythant**
A microsoft employee must use /azp run to validate using the pipelines below.

WARNING:
Comments made by azure-pipelines bot maybe inaccurate.
Please see pipeline link to verify that the build is being ran.

For status checks on the main branch, please use TransportPackage-Foundation-PR
(https://microsoft.visualstudio.com/ProjectReunion/_build?definitionId=81063&_a=summary)
and run the build against your PR branch with the default parameters.

### CODE REVIEW

File: tools/DevCheck/DevCheck.ps1 [hunk 1]
```diff
@@ -771,10 +771,10 @@ function Repair-DevTestPfx
 
     # -CertPassword <password> is a required parameter for this work
     $password = ''
-    $password_plaintext = $null
+    $passwordText = $null
```
Comment (codendone) [inline tools/DevCheck/DevCheck.ps1:774]: >$passwordText = $null [](http://example.com/codeflow?start=4&length=21)

Unused?

Comment (kythant) [reply tools/DevCheck/DevCheck.ps1:774]: Oh yes. Thanks for catching that.

**Pull request merged**